src/surbhi/ClassB.java

```diff
@@ -0,0 +1,18 @@
+package surbhi;
+
+public class ClassB {
+
+	void m2() {
+		System.out.println("m3");
```

Comment:
printing statements are wrong. you are in m2 method so why should we print "m3". I can see this mistake everywhere.